src/LazyOperations/ResetMap.jl

```diff
@@ -142,18 +144,22 @@ reset map.
 The vector contains the reset value for all reset dimensions, and is zero for
 all other dimensions.
 """
-function get_b(rm::ResetMap{N}) where {N<:Real}
-    return _get_b_from_dictionary(rm.resets, dim(rm))
+function vector(rm::ResetMap{N}) where {N<:Real}
+    return _vector_from_dictionary(rm.resets, dim(rm))
 end
 
-function _get_b_from_dictionary(dict::Dict{Int, N}, n::Int) where {N<:Real}
+function _vector_from_dictionary(dict::Dict{Int, N}, n::Int) where {N<:Real}
     b = sparsevec(Int[], N[], n)
     for (i, val) in dict
         b[i] = val
     end
     return b
 end
 
+function set(rm::ResetMap)
```

Comment:
should we `@inline` this?

Reply:
AFAIK Julia does this automatically for such obvious getter functions.